schema.js

```diff
@@ -0,0 +1,61 @@
+module.exports.userSchema = {
```

Comment:
Up here I suggest extracting a few repeated variables, and making use of them below:

```suggestion
const dateSchema = { bsonType: "Date", type: "string" };
const objectIdSchema = {
  bsonType: "ObjectId",
  type: "string",
  pattern: "[a-fA-F0-9]{24}"
};

module.exports.userSchema = {
```